definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-resp-sol-can-complete-aos-prod.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "LiveFrom": "18/07/2019",
+        "CaseTypeID": "DIVORCE",
+        "CaseEventID": "solicitorDraftAOS",
+        "UserRole": "[RESPSOLICITOR]",
+        "CRUD": "R"
```

Comment:
Should prod be "CRU" since in the core file it was "CRU"?

Reply:
@yarpo / @Dizrupto, that's actually the decision we've made.
We're going to "downgrade" this to be an "R" in production as of now. Otherwise, this will go live when share-a-case goes live (not what we want).
That's because the only thing stopping this event from being triggered in prod is the fact that the resp-solicitor role can't be assigned at the moment (but it will when share-a-case goes live)